manifests/kustomize/base/cache-deployer/cluster-scoped/cache-deployer-clusterrole.yaml

```diff
@@ -25,3 +25,11 @@ rules:
   - get
   - list
   - patch
+- apiGroups:
+  - certificates.k8s.io
```

Comment:
I was planning to fix this, would you mind also adding to mkp manifest?

Reply:
Here is the PR: https://github.com/kubeflow/pipelines/pull/4385